pagopa/io-app
# chore(EU Covid Certificate): [IAGP-9] Add getCertificate API network logic

## Short description
This PR add the `getCertificate` API

https://user-images.githubusercontent.com/822471/120338752-d23db400-c2f4-11eb-9728-504a56b4de38.mov


## changes
- create a dedicated backend client
- handle the backend response
- done some refactoring: moved repeated implementation of `tokenHeaderProducer` & `withBearerToken` in a dedicated file

## How to test
- make sure to have the last commit from master of [io-dev-server](https://github.com/pagopa/io-dev-api-server)
- if you want customize `getCertificate` API response, edit [this file](https://github.com/pagopa/io-dev-api-server/blob/90f00cf00ea860c1832eca39626006d10edd0768/src/routers/features/eu_covid_cert/index.ts#L25)
